url_shortener/cmd/httpload/main.go

```diff
@@ -0,0 +1,207 @@
+package main
+
+import (
+	"fmt"
+	"log"
+	"os"
+	"sort"
+	"strings"
+)
+
+const (
+	exitCodeOk    = 0
+	exitCodeError = 1
+)
+
+func computeStatuses() (statuses map[int]int, err error) {
+	statuses = make(map[int]int)
+
+	statuses[200] = 198
+	statuses[404] = 2
+
+	return statuses, err
+}
+
+func dumpStatuses(statuses map[int]int) string {
+	dumpBuilder := &strings.Builder{}
+
+	keys := make([]int, 0, len(statuses))
+
+	for key := range statuses {
+		keys = append(keys, key)
+	}
+
+	sort.Ints(keys)
+
+	for _, key := range keys {
+		code := float64(key)
+		count := float64(statuses[key])
+		fmt.Fprintf(dumpBuilder, "%s\n", fmt.Sprintf("[%3.f] %12.f response(s)", code, count))
+	}
+
+	return dumpBuilder.String()
+}
+
+func computeTimings() (timings []float64, err error) {
+	timings = make([]float64, 0, 512)
+
+	timings = append(timings, 0.0003)
+	timings = append(timings, 0.0054)
+	timings = append(timings, 0.0024)
+	timings = append(timings, 0.0295)
+
+	return timings, err
```

Comment:
`err` is not used!

Reply:
Same comment as of `computeStatuses` applies.